.github/workflows/tav-command.yml

```diff
@@ -0,0 +1,83 @@
+name: tav-command
+
+on:
+  issue_comment:
+    types: [created]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  command-validation:
+    if: github.event.issue.pull_request && startsWith(github.event.comment.body, '/test tav')
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    outputs:
+      versions: ${{ steps.generate-matrix.outputs.versions }}
+      modules: ${{ steps.generate-matrix.outputs.modules }}
+      pull-request: ${{ steps.generate-matrix.outputs.pull-request }}
+    steps:
+    - name: Validate github comment
+      uses: elastic/apm-pipeline-library/.github/actions/validate-github-comment@current
+
+    - name: Transform comment to the supported matrix
+      uses: actions/github-script@v6
+      with:
+        script: |
+          core.exportVariable('PR_NUMBER', context.payload.issue.number)
```

Comment:
At the moment a placeholder, but it might be the place where the GitHub command is parsed and the list of modules or node.js versions are filtered.